### DISCUSSION TITLE
display:block instead of display:grid

### DISCUSSION THREAD

**Discussion by wirmachenbunt** [cables-gl/cables_docs#729] [Answers & Questions] [answered]
i'm trying to get a css grid working...and it does not.
in my css i say display:grid; but when i inspect the css in dev tools there is display:block
where is this coming from ?  this breaks the css grid

<img width="1685" alt="image" src="https://github.com/cables-gl/cables_docs/assets/1138673/8e944091-a7ed-41b7-afb1-2f7d1a95b9db">

**Comment by wirmachenbunt** [comment]
ok making it !important does work.
would be cool if there is no hidden css added to div elements or at least some info about that would be helpful

**Comment by pandrr** [comment, marked as answer]
the visibility toggle of a divelement currently uses block when visible

on dev this is now "inherit" which should solve this problem  and not be breaking because block is the default for div elements

**Reply by wirmachenbunt** [reply]
ok, thanks
